docs/guide/personalization.md

```diff
@@ -36,17 +36,17 @@ $bundles = [
 
 ### 3. Import additional routing
 
-Import additional routing by adding the following lines to your `routing.yml` file:
+Import additional routing by adding the following lines to your `routing.yaml` file:
 
 ``` yaml
 recommendationBundleRestRoutes:
-    resource: '@EzSystemsRecommendationBundle/Resources/config/routing_rest.yml'
+    resource: '@EzSystemsRecommendationBundle/Resources/config/routing_rest.yaml'
     prefix: '%ezpublish_rest.path_prefix%'
 ```
 
 !!! note "Legacy support"
 
-    Legacy support is disabled by default. To enable the legacy search engine (requires `ezpublish-kernel` bundle) copy these service definitions to your `app/config/services.yml` file and uncomment them:
+    Legacy support is disabled by default. To enable the legacy search engine (requires `ezpublish-kernel` bundle) copy these service definitions to your `app/config/services.yaml` file and uncomment them:
```

Comment:
`config/services.yaml`

Reply:
Personalization page will be verified again in https://github.com/ezsystems/developer-documentation/pull/622